Sources/FeaturevisorSDK/Instance+Refresh.swift

```diff
@@ -46,6 +46,8 @@ extension FeaturevisorInstance {
 
                 case .failure(let error):
                     self.logger.error("failed to refresh datafile", ["error": error])
+
+                    self.emitter.emit(.error, error)
```

Comment:
for emitting errors, do you think it would be better if we have a signature like this:

```swift
self.emitter.emit(.error, ErrorType.someKey, errorObject)
```

that way, we will both have the full error object, and also the error type as a string, allowing users to filter errors out more conveniently:

```
// this is an example in JavaScript

f.onError(function (errorType, error) {
  if (errorType === ErrorType.datafileFetch) {
    // an error happened when fetching datafile
    console.log(error);
  }
});
```